notes/2023/2023-06-29.md

```diff
@@ -0,0 +1,60 @@
+# 2023-06-29 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2023-06-29-transcript.md`](2023-06-29-transcript.md)
+
+## Attending
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### [Where we are on the integration with outside plugins/configs listed on this issue?](https://github.com/eslint/eslint/issues/13481)
+
+> @nzakas: On my end, I've opened PRs for each of the projects. None have been merged yet.
+
+Status:
+- [x] `eslint-plugin-n` now provides flat config exports
+- [ ] `typescript-eslint` PR opened; waiting for Nx to get updated to use flat config
+- [ ] `eslint-config-standard` PR opened
+- [ ] `eslint-plugin-vue` PR opened
+- [ ] `eslint-plugin-react` PR to be submitted by @mdjermanovic
```

Comment:
eslint-plugin-react has added its support in v7.32.0:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/CHANGELOG.md#added-1

Reply:
eslint-plugin-react started exporting flat configs in v7.32.0 but they are not using flat config in the repo. That's what @mdjermanovic is going to send a PR for